main.go

```diff
@@ -2,36 +2,85 @@ package main
 
 import (
 	"fmt"
+	"log/slog"
 	"net/http"
 	"os"
 	"time"
 
 	metalgo "github.com/metal-stack/metal-go"
 
-	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
-	klog "k8s.io/klog/v2"
+)
+
+const (
+	fetchInterval = 120 * time.Second // time to sleep after every metrics fetch
```

Comment:
We need to discuss... before this the scrape interval was 30s, which was sometimes insufficient to figure out the metrics. To me, I think that every two minutes is actually enough?

Reply:
Now made this configurable and lowered to 90s instead of 120s.